fl4health/preprocessing/autoencoders/dim_reduction.py

```diff
@@ -0,0 +1,84 @@
+from abc import ABC
+from pathlib import Path
+
+import torch
+
+
+class Processing(ABC):
```

Comment:
I would say processing is a bit too generic a name since we're specifically loading some kind of auto-encoder. I know it's in the `autoencoders` folder, but if you see it in code without referencing the important it isn't completely obvious that it's autoencoder related? Perhaps `AutoEncoderProcessing`?